sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/BearerTokenAuthenticationPolicy.java

```diff
@@ -1,45 +1,34 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.v2.core.http.policy;
+package io.clientcore.core.http.pipeline;
 
-import com.azure.v2.core.credentials.AccessToken;
-import com.azure.v2.core.credentials.TokenCredential;
-import com.azure.v2.core.credentials.TokenRequestContext;
-import com.azure.v2.core.implementation.AccessTokenCache;
+import io.clientcore.core.credentials.AccessToken;
+import io.clientcore.core.credentials.TokenCredential;
+import io.clientcore.core.credentials.TokenRequestContext;
+import io.clientcore.core.implementation.AccessTokenCache;
 import io.clientcore.core.http.models.HttpHeaderName;
 import io.clientcore.core.http.models.HttpHeaders;
 import io.clientcore.core.http.models.HttpRequest;
 import io.clientcore.core.http.models.Response;
-import io.clientcore.core.http.pipeline.HttpCredentialPolicy;
-import io.clientcore.core.http.pipeline.HttpPipeline;
-import io.clientcore.core.http.pipeline.HttpPipelineNextPolicy;
-import io.clientcore.core.http.pipeline.HttpPipelinePolicy;
 import io.clientcore.core.instrumentation.logging.ClientLogger;
+
 import java.io.IOException;
 import java.util.Objects;
 
 /**
  * <p>The {@code BearerTokenAuthenticationPolicy} class is an implementation of the {@link HttpPipelinePolicy} interface.
```

Comment:
It looks like it is an implementation of `HttpCredentialPolicy` now, rather than `HttpPipelinePolicy`, and that is not an interface.